tests/it/fixtures/fcos-stream.json

```diff
@@ -184,6 +184,14 @@
                 }
             },
             "images": {
+                "aliyun":{
+                    "regions": {
+                        "us-east-1": {
```

Comment:
are there more regions than `us-east-1`? Does it matter if we don't list them here?

Reply:
I believe we are just testing with one region as of now. @bgilbert Kindly confirm if more regions need to be added.

Reply:
There are more regions, but this is just a test fixture, so I think this is okay.